src/utils/rescueFile.ts

```diff
@@ -6,6 +6,12 @@ import {
 } from "@scure/bip39";
 import { wordlist } from "@scure/bip39/wordlists/english";
 
+export enum Errors {
+    InvalidBackupFile = "invalid file",
```

Comment:
```suggestion
    InvalidFile = "invalid file",
```
Because we also use this for the rescue file now

Reply:
@michael1011 changed!